components/sections/quest-log.module.css

```
@@ -0,0 +1,327 @@
+}
+
+.markerPulse {
+  position: absolute;
+  inset: -6px;
+  border-radius: 10px;
+  box-shadow: 0 0 0 0 rgba(232, 182, 76, 0.45);
+  animation: markerPulse 2.8s ease-out infinite;
+  pointer-events: none;
+}
+
+@keyframes markerPulse {
```

Stylelint (17.14.1)

[error] Expected keyframe name "markerPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)